### ISSUE TITLE
Upgrade simple form to use official bootstrap template

### ISSUE THREAD

**Issue by murny** [ualbertalib/jupiter#671]
Looks like Simple Form 4.0 has an official bootstrap v4.1 template. Let's use this instead of ours.

More info: https://simple-form-bootstrap4.herokuapp.com/

**Comment by mbarnett** [comment]
Is this still relevant, @murny ?

**Comment by murny** [comment]
Yeah I believe it is. We have this initializer which we wrote (or taken from somewhere) and we can instead use the one straight out of SimpleForm now.

Essentially replace:
https://github.com/ualbertalib/jupiter/blob/integration_postmigration/config/initializers/simple_form_bootstrap.rb

With:
https://github.com/rafaelfranca/simple_form-bootstrap/blob/master/config/initializers/simple_form_bootstrap.rb

(and hopefully it all works)

**Comment by mbarnett** [comment]
Gotcha
